src/styles/blocks/buy.scss

```diff
@@ -0,0 +1,19 @@
+.buy {
```

Comment:
no needs to create so many scss file. You need only
card.scss
stars.scss
header.scss
nav.scss
variables.scss
main.scss